config/config.md

```diff
@@ -132,7 +132,7 @@ The following config option are provided by the OpenHIM. All of these options ha
     "orchestrationTSBufferMillis": 100
   },
   "statsd": {
-    // Rather use a statsd server for reporting metrics
+    // Report application metrics and transaction metrics to a statsd server
```

Comment:
aren't we getting rid of statsd though? or just api/statsd? I must admit to being a little confused how it's all fitting together! :) is [src/stats](https://github.com/jembi/openhim-core-js/blob/2cd15ba8d9e5e2147fc7427bb2d64f455d7cf004/src/stats.coffee) different to the metrics?